oleesir/diaryReload
# feat(signup): implement signup page

-implement signup page

[Finishes #160870200]

What does this PR do?
This pull request adds a signup template to the app

 Description of Task to be completed?
add signup template

How should this be manually tested?
Navigate to the UI folder and open signup.html in your browser
You will see a signup template

What are the relevant pivotal tracker stories?
#160870200

#### Any background context you want to add?
N/A
<img width="1280" alt="screen shot 2018-09-29 at 11 54 59 pm" src="https://user-images.githubusercontent.com/17679511/46251152-25c01b80-c443-11e8-9621-9d42ce1f9ca0.png">


#### Screenshots
